packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.ts

```diff
@@ -144,6 +144,19 @@ export default ({ api, coreSagas, networks }) => {
     )
   }
 
+  const defineCowboysCampaignGoal = function* (search) {
+    // /#/open/cowboys?token_id=456
```

Comment:
How about something like `/#/open/referrals` which will open the referral flyout. Then in the `useEffect()` we check for the user tag `COWBOYS_2022` to display the proper branding?

Reply:
yeah I was unsure how we want to have this from marketing perspective